getpaid_payu/client.py

```diff
@@ -97,7 +97,7 @@ def _normalize(cls, data: Union[dict, list]):
         data = deepcopy(data)
         if hasattr(data, "items"):
             return {
-                k: Decimal(v) / 100
+                k: cls._convert(Decimal(v) / 100)
```

Comment:
Why? This will have the same effect as `int(Decimal(v))` because `_convert` multiplies by 100. This method is intended to turn PayU's amount format (where the amount is multiplied by 100; 1€ is 100) to common (with cents/groszes).

Reply:
I was my first think (int) but result was wrong. Check `int(19.99)` return `19`.
Payu API require  1999 and convert it to 19.99 PLN/EUR

Reply:
OK but this method **normalizes** the values - it turns them from "cent" to normal notation, so from 1999 to 19.99. By introducing `_convert` you do nothing as `*100` from `_convert` and `/100` cancel each other out. So instead of turning 1999 to 19.99 you just leave it as it is. note that this is a dict comprehension iterating over `data.items()` with extra recursion to normalize each nested dict.